### PR TITLE
Tech/1795/enforce line endings 3

### PR DESCRIPTION
**Pull request by BridgeAR**
This contains the editorconfig, the gitattributes file and an improved changelog while also fixing prettier issue detection on our CI runs and makes sure files in python are saved with \n instead of \r\n.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```
Comment (ce-bo) [inline .editorconfig:6]: I am not sure. Do we use indent_size of 4 at the moment?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+quote_type = double
+
+[*.kt]
+indent_size = unset
+quote_type = unset
```
Comment (ce-bo) [inline .editorconfig:14]: What was the reason for unsetting those values? :)

File: .editorconfig (hunk 1 above)
Comment (BridgeAR) [reply .editorconfig:6]: We use ktlint for kotlin files and the indentation is not 2 but partially 3 and 4.

File: .editorconfig (hunk 2 above)
Comment (BridgeAR) [reply .editorconfig:14]: I was not sure anymore if the quotes are different due to ktlint. It's not necessary to define it here, as ktlint is going to complain about them anyway, if they are wrong.

**Review by ce-bo — APPROVED** [review]

**Pull request merged**